src/scripts/librarymenu.js

```diff
@@ -962,7 +962,7 @@ define(["dom", "layoutManager", "inputManager", "connectionManager", "events", "
     });
     events.on(playbackManager, "playerchange", updateCastIcon);
     events.on(syncplayManager, "SyncplayEnabled", updateSyncplayIcon);
-    events.on(syncplayManager, "SyncplayError", updateSyncplayErrorIcon);
+    events.on(syncplayManager, "SyncplayProblem", updateSyncplayStatusIcon);
```

Comment:
what's the difference between `updateSyncplayIcon` and `updateSyncplayStatusIcon`?

Reply:
One is called when SyncPlay is enabled/disabled (user joins or leaves a group), the other is called when the media is being synced (while SyncPlay is enabled).

I'll try to give them a better naming. Or should I join them together?